docs/security/how-to-use-authorizer.md

```diff
@@ -0,0 +1,30 @@
+---
+id: what-is-authorizer
+title: What is Authorizer?
+sidebar_label: What is Authorizer?
+---
+
+The `Authorizer` contains a single `Authorize` method, which is invoked for each incoming API call. `Authorize` receives information about the API call, along with the role and permission claims of the caller.
+
+Authorizer allows for a wide range of authorization logic, including call target, role/permissions claims, and other data available to the system.
+
+## Configuration
+
+The following arguments must be passed to the Authorizer:
+
+- `context.Context`: General context of the call.
+- `authorization.Claims`: Claims about the roles assigned to the caller. Its intended use is [described below](#claims).
+- `authorization.CallTarget`: Target of the API call.
+
+Authorizer then returns one of two decisions:
+
+- `DecisionDeny`: the requested API call is not invoked and an error is returned to the caller.
+- `DecisionAllow`: the requested API call is invoked.
+
+Configure your `Authorizer` when you start the server via the `temporal.WithAuthorizer` [server option](/server/options/#withauthorizer).
+
+If an `Authorizer` is not set in the server options, Temporal uses the `nopAuthority` authorizer that unconditionally allows all API calls to pass through.
```

Comment:
This is a pretty big flag, and we should make it very clear that the default behavior is "all API calls pass". If an end-user accidentally breaks their `Authorizer` with something as simple as a typo, it could result in all traffic being allowed, which nobody will notice because nothing is breaking.

Ideally, I'd like for the default authorizer to *deny* everything, and if you want to use the `nopAuthority` then it must be set explicitly. For now, we should at least make this called out in big, flashy, scary, letters. Where's the `<blink>` tag when you need it‽

Reply:
Good point. I was wondering why the heck it defaults like that, and there really should be emphasis on it.